Formula/circleci.rb

```diff
@@ -35,13 +36,14 @@ def install
 
   test do
     # assert basic script execution
```

Comment:
How about 
```suggestion
    ENV["CIRCLECI_CLI_TELEMETRY_OPTOUT"] = "1"
    # assert basic script execution
```

Reply:
Sure, thanks for the suggestion. I don't know ruby at all